### PR TITLE
Feature/remove youtube

### PR DESCRIPTION
**Pull request by jdmedlock**
This PR fixes the issues & tasks reported in #5 & #6.

Issue #5 Removes the HTML that displays the link to the Chingu YouTube channel.
Issue #6 Adds a word count next to the input search box so the user can see how many words have been entered. To do this a new function, `countWords` was added to `+page.svelte`.

### CODE REVIEW

**Review by alcb1310 — CHANGES_REQUESTED** [review]
A console log is still present when I call the api, there should not be any console.logs

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,121 +1,155 @@
...
 
-  const validateSearchWord = () => {
-    isError = false
-    if (searchFor === null || searchFor === undefined || searchFor === '') {
-      isError = true
-      errorMessage = ERR_INVALID_SEARCH_TEXT
-      return
-    }
+	// Count the number of words in the search box
+	const countWords = () => {
```
Comment (alcb1310) [inline src/routes/+page.svelte:26]: You should use a string function searchFor.split(" ") so you can check for words, this function will convert the different words of the string into items in an array and that way you can use the length of the array to count the words

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -1,121 +1,155 @@
...
+		wordCount = 0
+		searchFor = searchFor.trim()
+		console.log(searchFor)
```
Comment (alcb1310) [inline src/routes/+page.svelte]: Please remove the console.log before merging

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -94,7 +83,7 @@
 	<div class="mt-4 ml-8 text-lg">
 		<div class="mt-4">
 			<label for="searchFor">
-				Search for the definition of:
+				Search for the definite of:
```
Comment (alcb1310) [inline src/routes/+page.svelte]: Please change the `definite` word for `definition` it will be more descriptive of what is asked for the user

**Comment by jdmedlock** [comment]
> A console log is still present when I call the api, there should not be any console.logs

Thanks. `console.log` statements have been removed

**Review by alcb1310 — CHANGES_REQUESTED** [review]
Look at this comments inside the files please

File: src/routes/+page.svelte (hunk 1 above)
Comment (jdmedlock) [reply src/routes/+page.svelte:26]: Great idea. This should also be more performant. Issue resolve.

File: src/routes/+page.svelte (hunk 2 above)
Comment (jdmedlock) [reply src/routes/+page.svelte]: Thanks. `console.log` removed.

File: src/routes/+page.svelte (hunk 3 above)
Comment (jdmedlock) [reply src/routes/+page.svelte]: This spelling error has been resolved.

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -1,121 +1,144 @@
...
-    isError = false
-    if (searchFor === null || searchFor === undefined || searchFor === '') {
-      isError = true
-      errorMessage = ERR_INVALID_SEARCH_TEXT
-      return
-    }
+	// Count the number of words in the search box
+	const countWords = () => {
+		searchFor = searchFor.trim()
+		wordCount = searchFor.split(' ').length
```
Comment (alcb1310) [inline src/routes/+page.svelte:28]: Nice!! I like it that you made it a one liner

**Review by alcb1310 — APPROVED** [review]
Great, I think everything is resolved now

File: src/routes/+page.svelte (hunk 4 above)
Comment (alcb1310) [reply src/routes/+page.svelte:28]: Please resolve this conversation when you see it

Comment (jdmedlock) [reply src/routes/+page.svelte:28]: Thanks!

**Comment by jdmedlock** [comment]
PR approved. Ready to be promoted to production

**Pull request merged**